public/index.js

```diff
@@ -104,12 +104,13 @@ class App extends React.Component {
     )
   }
 
-  debugOnChange = (newValue) => {
+  handleChange = (newValue) => {
```

Comment:
Did eslint ask you to do this?

Reply:
Yes. It's `react/jsx-handler-names` of `eslint-plugin-react`.